src/actions/sources/createPrettySource.js

```diff
@@ -13,6 +13,28 @@ export function createPrettySource(sourceId) {
     const url = getPrettySourceURL(source.get("url"));
     const id = await sourceMaps.generatedToOriginalId(sourceId, url);
 
+    const prettySource = {
+      url,
+      id,
+      isPrettyPrinted: true,
+      text: "Loading...",
+      contentType: "text/javascript",
+      frames: [],
+      loadedState: "loadeding"
+    };
+    dispatch({
+      type: "ADD_SOURCE",
+      source: prettySource
+    });
```

Comment:
this can be one line 

`  dispatch({    type: "ADD_SOURCE",      source: prettySource }); `